packages/angular/cli/commands/deploy-long.md

```diff
@@ -1,9 +1,38 @@
-The `ng deploy` command is a shortcut for:
+The command is a shortcut for:
 
-```
+<code-example language="bash">
 ng run [PROJECT_NAME]:deploy
-```
+</code-example>
 
 It takes an optional project name, as specified in the `projects` section of the `angular.json` workspace configuration file.
+When a project name is not supplied, executes the `deploy` builder for the default project.
 
-When a project name is not supplied, the CLI will execute the `deploy` builder for the default project.
+The deploy builder defined in a deployment platform package is automatically added to a project's configuration when you add the package to the project.
```

Comment:
Can we update it to something like:

To use the `ng deploy` command `ng add` a package that implements deployment capabilities to your favorite platform. The `ng-add` schematics of the package will automatically update your workspace configuration, adding a deployment CLI builder.

Reply:
Also adding link to CLI Builder guide.